cmd/nomos/status/client.go

```diff
@@ -140,6 +143,16 @@ func (c *ClusterClient) resourceGroups(ctx context.Context, ns string, nsAndName
 func (c *ClusterClient) clusterStatus(ctx context.Context, cluster, namespace string) *ClusterState {
 	cs := &ClusterState{Ref: cluster}
 	var err error
```

Comment:
nit: `var err error` is not needed as you declared `isOss, err :=`.

Reply:
removed!